content/posts/performance_presentation.Rmd

```diff
@@ -0,0 +1,93 @@
+
+***To address this, please let us introduce the ***  [**`performance`**](https://github.com/easystats/performance) ***package!***
+
+
+## performance
+
+[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life *easier* (currently WIP). This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), as well as [indices of model performance](https://github.com/easystats/performance).
+
+
+The goal of [`performance`](https://github.com/easystats/performance) is to provide lightweight tools to **assess and check the quality of your model**. It includes functions such as [`R2`](https://easystats.github.io/performance/reference/r2.html) for many models (including logistic, mixed and Bayesian models), [`ICC`](https://easystats.github.io/performance/reference/icc.html) or helpers to check [`convergence`](https://easystats.github.io/performance/reference/check_convergence.html), [`overdipsersion`](https://easystats.github.io/performance/reference/check_overdispersion.html) or [`zero-inflation`](https://easystats.github.io/performance/reference/check_zeroinflation.html). See the **list of functions** [here](https://easystats.github.io/performance/reference/index.html).
+
+`performance` can be installed as follows:
+
+```{r, eval = FALSE, message=FALSE, warning=FALSE}
+install.packages("performance")  # Install the package
+library(performance)  # Load it
+```
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(performance)
+```
+
+## Example
+
+Let's compute **all available** indices of performance appropriate for a given model. This can be done via the [**model_performance**](https://easystats.github.io/performance/reference/model_performance.html).
+
+
+### Mixed Models
+
+```{r message=FALSE, warning=FALSE}
+# Load the lme4 package
+library(lme4)
+
+# Fit a mixed model
+model <- lmer(Sepal.Width ~ Petal.Length + (1|Species), data = iris)
+
+# Compute performance indices
+model_performance(model)
+```
+
+
+### Bayesian Mixed Models
+
+```{r eval = FALSE, message=FALSE, warning=FALSE}
+# Load the rstanarm package
+library(rstanarm)
+
+# Fit a Bayesian mixed model
+model <- stan_glmer(Sepal.Width ~ Petal.Length + (1|Species), data = iris)
+
+# Compute performance indices
+model_performance(model)
+```
+```{r message=FALSE, warning=FALSE, echo=FALSE}
+library(rstanarm)
+junk <- capture.output(model <- stan_glmer(Sepal.Width ~ Petal.Length + (1|Species), data = iris))
+model_performance(model)
+```
+
+
+
+More details about `performance`'s features are comming soon, stay tuned ;)
+
+- **Don't forget to check out the** [**documentation here**](https://easystats.github.io/performance/) **for more!**
```

Comment:
I would bump this up a line (i.e., over "More details about..." in line 82), as line 82 feels like a conclusion, but this line feels like a reminder that should come before the conclusion. Just a thought.

Reply:
you're right!